float64 occ
float32[] acc
float64[] a
float64 statio
float64[] phi
float64[] gamma
int32 n
int32 visits
